requirements/edx/base.txt

```diff
@@ -136,6 +136,7 @@ freezegun==0.1.11
 lettuce==0.2.20
 mock-django==0.6.6
 mock==1.0.1
+moto==0.3.1
```

Comment:
@feanil @singingwolfboy This is a new testing library that I'm using to test image upload and download from S3.  We used this in ORA2.

I wasn't able to use the most recent version of the library, because it requires a newer version of `boto` than what we're currently using.